### PR TITLE
Stud 249 UI for active sale

### PR DESCRIPTION
**Pull request by escobarjonatan**
This includes:
- Create sale UI (minus summary modal)
- Create/End sale endpoint additions
- Active sale UI with partial endpoint logic to cancel sale (currently manual process)
- Pinging for sale endpoint on sale end/start 

https://github.com/user-attachments/assets/08fb9f37-3805-4709-bb82-063d6d6cd936

### CODE REVIEW

File: apps/studio/src/api/utils.ts [hunk 1]
```diff
@@ -105,18 +105,12 @@ export const getAuthHeaders = (api: BaseQueryApi) => {
...
   const action = recaptchaEndpointActionMap[endpoint] || endpoint;
 
-  if (!isLoggedIn) {
-    return {
-      "g-recaptcha-platform": "Web",
-      "g-recaptcha-token": await executeRecaptcha(action),
-    };
-  }
-
-  return {};
```
Comment (escobarjonatan) [inline apps/studio/src/api/utils.ts:119]: @scandycuz Having this prevented us from calling the get sales marketplace endpoint in studio. Let me know your thoughts on removing it

File: apps/studio/src/App.tsx [hunk 2]
```diff
@@ -50,6 +52,8 @@ const App = () => {
             <ProgressBarModal />
             <UpdateWalletAddressModal />
             <WalletEnvMismatchModal />
+            <PingSaleStart />
+            <PingSaleEnd />
```
Comment (escobarjonatan) [inline apps/studio/src/App.tsx:56]: These components will handle polling for new sales when one gets created/removed in order to display an accurate UI state of the sale

File: apps/studio/src/api/utils.ts (hunk 1 above)
Comment (scandycuz) [reply apps/studio/src/api/utils.ts:119]: Should be fine to send the headers for all requests

**Review by scandycuz — APPROVED** [review]
LGTM!

**Pull request merged**